### ISSUE TITLE
Description of value to set property is misleading

### ISSUE THREAD

**Issue by robinhultman** [BizTalkComponents/ContextRegExReplace#4]
The description  of Value to Set property indicates that the matched value is replaced by a hard coded value which it is not. From MSDN:
_For example, the replacement pattern a_${test}b inserts the string "a_" followed by the substring that is matched by the test capturing group, if any, followed by the string "b". The \* character is not recognized as a metacharacter within a replacement pattern._

**Comment by riha** [comment]
Yep. But the value doesn't have to be a RegEx but can of course also be a hard coded value. Turns out that this can be used in a number of ways and the documentation should probably include a number of different examples. The way we used it previously is that we looked for a match in a value using RegEx and in case of a match we replaced the value in another property with a hard coded value for further routing down the line (for example if the status contains "30" set "1" in property x).
